src/web/lib/components/PaginatedThemeSelector/index.scss

```diff
@@ -13,44 +13,63 @@
     margin: 0;
 
     button {
-      color: #000;
-      background-color: transparent;
+      background: transparent;
       border: 0;
+      color: transparent;
       cursor: pointer;
-      margin: 2px;
       height: 18px;
+      margin: 2px;
       overflow: hidden;
-      text-indent: 10px;
+      padding: 0;
+      position: relative;
       white-space: nowrap;
-      width: 10px;
+      width: 18px;
     }
 
     button.page {
-      text-indent: 0;
-
       &::before {
-        content: "\25cf";
+        content: "";
+        border-radius: 8px;
+        border: 2px solid var(--grey-40);
+        box-sizing: border-box;
+        height: 10px;
+        left: 4px;
+        position: absolute;
+        top: 4px;
+        width: 10px;
       }
     }
 
     button.current-page {
-      text-indent: 0;
-
       &::before {
-        content: "\25cb";
+        content: "";
+        border-radius: 8px;
+        background: var(--grey-60);
```

Comment:
> Or, possibly just using `<button class="page">` and `<button class="page current-page">` so that the `current-page` rule only has to tweak the `border` and `background` styles.

This is a good suggestion.